src/library/export/coverartcopyworker.cpp

```diff
@@ -9,7 +9,16 @@
 #include "util/safelywritablefile.h"
 
 void CoverArtCopyWorker::run() {
-    auto selectedCover = mixxx::FileAccess(mixxx::FileInfo(m_selectedCoverArtFilePath));
+    auto coverArtFileInfo = mixxx::FileInfo(m_selectedCoverArtFilePath);
+    auto selectedCoverFileAccess = mixxx::FileAccess(coverArtFileInfo);
+    if (!selectedCoverFileAccess.isReadable()) {
+        if (Sandbox::askForAccess(&coverArtFileInfo)) {
+            selectedCoverFileAccess = mixxx::FileAccess(coverArtFileInfo);
```

Comment:
We need also check for failure in the second path. if this code remains.

Reply:
So double check `isReadable` for both of the paths would work?